scripts/clone_template.py

```diff
@@ -187,11 +189,17 @@ def main(**kwargs):
         deploy_args['ssh_key'] = '{user_name}:{public_key}'.format(
             user_name=cred['ssh']['ssh-user'],
             public_key=cred['ssh']['public_key'])
+    elif provider_type == 'openshift':
+        trackerbot = api()
+        raw_tags = trackerbot.providertemplate().get(provider=kwargs['provider'],
+                                                     template=deploy_args['template'])['objects']
+        raw_tags = raw_tags[-1]['template'].get('custom_data', "{}")
+        deploy_args["tags"] = yaml.safe_load(raw_tags.replace("u'", '"').replace("'", '"'))['TAGS']
```

Comment:
What are you trying to replace here?  Can you add some inline comments for what you're intending to pull out/replace please?

Reply:
sure thing. each openshift appliance version stored in trackerbot as json data. It turned out that that json data is slightly screwed up by trackerbot api. So, these replacements just turn that data into format reable by yaml.